### ISSUE TITLE
Can't reference frontmatter if sending variable to partial

### ISSUE THREAD

**Issue by DusanMilko** [assemble/assemble#733]
I call partial like this:

```handlebars
{{> partial title="TITLE"}}
```

and my partial also has frontmatter

```handlebars
---
module: hero
---

<div class="{{module}}">
  {{title}}
</div>
```

The title comes in but the frontmatter (module variable) doesn't work

I see I can do:

```handlebars
---
module: hero
---

<div class="{{partial.module}}">
  {{title}}
</div>
```

is there a way around that, maybe like??

```handlebars
---
module: hero
---

<div class="{{this.module}}">
  {{title}}
</div>
```

??

**Comment by DusanMilko** [comment]
I see I can do:

```
---
module: hero
---

<div class="{{partial.module}}">
  {{title}}
</div>
```

is there a way around that, maybe like??

```
---
module: hero
---

<div class="{{this.module}}">
  {{title}}
</div>
```

**Comment by jonschlinkert** [comment]
I believe you need to reference the parent scope with `../module`. the handlebars docs are useful for this. 

also, you should be able to use middleware or custom plugins to completely customize how context is merged in the latest assemble (0.6.0 and up).
